src/utils/utils.tsx

```diff
@@ -31,13 +31,14 @@ export interface AssetObject extends ParentAssetObject {
 }
 
 interface ParentAssetObject extends ElementObject {
+  asset_number: string;
   hostname: string;
   rack_position: string;
   mac_addresses: { [port: string]: string };
   network_connections: Array<NetworkConnection>;
   power_connections: { [port: string]: PowerConnection };
-  owner?: string;
-  comment?: string;
+  owner: string;
+  comment: string;
```

Comment:
removing these as ` |undefined` fields in the frontend -> this is not and should not be direct mapping between a required/optional field in the backend